docs/src/components/Testimonials/TestimonialItem/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { PropsWithChildren } from 'react';
+import clsx from 'clsx';
+import styles from './styles.module.css';
+import QuoteIcon from './QuoteIcon';
+import { useColorMode } from '@docusaurus/theme-common';
+
+interface ImageProps {
+  alt: string;
+  src: string;
+}
+interface Props extends PropsWithChildren {
+  author: string;
+  company?: string;
+  link: string;
+  image: ImageProps;
+}
+
+const TestimonialItem = ({ author, company, image, link, children }: Props) => {
```

Comment:
Where are we putting these Testimonials?

Reply:
Like, where on the page? Above sponsors, but they will be there once we gather the testimonials in the future i suppose?

Reply:
Sure, I was just asking if we didn't forget about them 😄